Storage/src/Connection/Rest.php

```diff
@@ -302,6 +314,15 @@ public function insertObject(array $args = [])
             ]
         ];
 
+        // Passing the preconditions we want to extract out of arguments
+        // into our query params.
+        $preconditions = RetryTrait::$condIdempotentOps['objects.insert'];
```

Comment:
PHP has deprecated referencing static properties on traits:

```
PHP Deprecated:  Accessing static trait property RetryTrait::$condIdempotentOps is deprecated, it should only be accessed on a class using the trait in php shell code on line 1
```

However, since this class implements the trait, you can just use `self::$condIdempotentOps`